src/mpid/ch4/netmod/ucx/ucx_init.c

```diff
@@ -110,8 +110,8 @@ static int initial_address_exchange(MPIR_Comm * init_comm)
                               &MPIDI_UCX_AV(&MPIDIU_get_av(0, node_roots[i])).dest[0][0]);
             MPIDI_UCX_CHK_STATUS(ucx_status);
         }
-        MPIDU_bc_allgather(init_comm, MPIDI_UCX_global.ctx[0].if_address,
-                           (int) MPIDI_UCX_global.ctx[0].addrname_len, FALSE,
+        MPIDU_bc_allgather(init_comm, (void **) &MPIDI_UCX_global.ctx[0].if_address,
+                           (int *) &MPIDI_UCX_global.ctx[0].addrname_len, 1, FALSE,
                            (void **) &table, &rank_map, &recv_bc_len);
```

Comment:
Stale changes?

Reply:
Oops. I forgot about UCX. I'll fix that quickly.